apiary.apib

```diff
@@ -2838,6 +2838,68 @@ Update a milestone.
 
 + Response 204 (application/json;charset=utf-8)
 
+## Materials [/materials]
+
+Materials are added to a project and they affect the profitability. This information is only available for superusers and project decision makers.
```

Comment:
Q: Do we can them superusers to the outside world? I thought we were trying to get rid of that word.

Reply:
indeed, we call them `account administrators`

```suggestion
Materials are added to a project and they affect the profitability. This information is only available for account administrators and project decision makers.
```

Reply:
^ the above suggestion will not work, as this is compiled code